src/uu/chcon/chcon.md

```diff
@@ -0,0 +1,11 @@
+# chcon
+
+```
+chcon [OPTION]... CONTEXT FILE...
+chcon [OPTION]... [-u USER] [-r ROLE] [-l RANGE] [-t TYPE] FILE...
+chcon [OPTION]... --reference=RFILE FILE...
```

Comment:
the spell check is complaining here

Reply:
Added an ignore at the beginning of the file 85567d5
However I am asking myself whether `RFILE` should not be added to this list as it's not the only place where it's used. https://github.com/uutils/coreutils/blob/f5a9ffe52fb40c045684ea95129f6020e04bfff2/.vscode/cspell.dictionaries/jargon.wordlist.txt
What do you think ?

Reply:
Yeah that sounds good!